client/witness.go

```diff
@@ -10,7 +10,7 @@ import (
 
 // Witness creates change objects to witness (i.e. re-sign) the given
 // roles on the next publish. One change is created per role
-func (r *NotaryRepository) Witness(roles ...string) ([]string, error) {
+func (r *NotaryRepository) Witness(roles ...data.RoleName) ([]string, error) {
```

Comment:
Would it make sense to return a list of `data.RoleName` instead of strings here?